test/integ-new/main_new_test.go

```diff
@@ -303,6 +304,38 @@ func TestCertRotatorKeyRotation(t *testing.T) {
 				3: kmspb.CryptoKeyVersion_ENABLED,
 			})
 	})
+
+	t.Run("invalid_primary", func(t *testing.T) {
+		// Set primary to a version that doesn't exist
```

Comment:
My memory is failing me, which means it's something we should document better - What's the purpose of setting primary to a non-existing version? I see your comment below but I can't understand it.

Reply:
Maybe a better edge case is when primary label is missing.

Reply:
updated to remove the label